js/Discourse.js

```diff
@@ -219,7 +220,7 @@ class Discourse extends React.Component {
       let params = this.parseURLparameters(event.url);
       let site = this._siteManager.activeSite;
 
-      if (Platform.OS === 'ios') {
+      if (Platform.OS === 'ios' && Settings.get('external_links_svc')) {
```

Comment:
What's that? 🙂

Reply:
It's an iOS-only setting. Some users prefer to use the Safari in-app-browser (instead of the webview), and it makes sense to only dismiss SafariView when that setting is enabled.